content/post/glue-work.md

```diff
@@ -0,0 +1,83 @@
+
+If you don’t have any women on your analytics team, well, [think about that](https://www.forbes.com/sites/metabrown/2017/03/30/she-asked-for-thoughts-on-women-in-analytics-heres-what-i-said/#1b4c7e35a5df) the next time you’re hiring. But in male-only groups, the same men tend to [volunteer](https://hbr.org/2018/07/why-women-volunteer-for-tasks-that-dont-lead-to-promotions) time after time for non-promotable tasks. So regardless of the gender makeup of your team, a small number of people on your team are probably doing this work without being rewarded for it.
+
+### The Meta-Story
+Arguably, analytics itself is glue within many companies. In most companies, the analytics team is not responsible for a revenue target. The team may not own any OKRs. The impact of analytics is subtler. Good analysts are multipliers, enabling the company to identify and focus on areas where changes can have the most impact. We don’t build features, we don’t close deals, but we shape the processes, tools, and cultures that enable companies to be smarter and faster across the board.
+
+Yet, there is often confusion about [what analysts are](https://hbr.org/2018/12/what-great-data-analysts-do-and-why-every-organization-needs-them) - they just write reports, right? Data science has the opposite problem - folks on the outside think it’s really exciting, but the reality is that it’s exciting occasionally, and boring a lot. In either role, when you tell someone that the “simple” request they asked for is really multiple days or weeks of work because the data isn't yet available, isn't well-defined, or needs significant transformation work, their will eyes glaze over.
+
+These disconnects represent a misunderstanding between the company as a whole and the analytics team about what is valuable, and what is valued. Clearly identifying and communicating why different types of work is valued within the analytics team sets the stage to more clearly communicate the value of the analytics team itself to the rest of the company.
+
+## So What Do We Do About It?
+The problem of glue work is fixable. If your team is doing work that helps improve the likelihood of success on projects, that work needs to be explicitly valued.
+
+### Managers
+If you are in a position to drive systematic change in your organization, you need to:
+- Figure out what your team is doing that is not written in their job descriptions. Watch them, and ask them - about themselves and their teammates. Separate the list into work that requires an analyst’s skills and improves the output of the analytics team, and work that doesn’t.
+- Work that is truly valuable to your team’s mandate? Write clear career ladders that reward what needs to be done - including glue work. This might include documentation, relationship-building, onboarding new team members, and building subject matter expertise. I have a lot more to say on this - enough to fill another post, coming soon.
```

Comment:
It might be clearer to phrase the bullets the bullets that start with questions like "If the work is truly valuable to your team's mandate..."

Alternatively if you want to keep the question format, maybe write the question in italics to emphasize that it frames the bullet point, but isn't part of the suggestion.